apps/trading/components/welcome-dialog/onboarding-dialog.tsx

```diff
@@ -0,0 +1,233 @@
+import { t } from '@vegaprotocol/i18n';
+import { GetStarted } from './get-started';
+import { TradingButton } from '@vegaprotocol/ui-toolkit';
+import { useNavigate } from 'react-router-dom';
+import { Links, Routes } from '../../pages/client-router';
+import { useLocalStorage } from '@vegaprotocol/react-helpers';
+import * as constants from '../constants';
+import { useEnvironment } from '@vegaprotocol/environment';
+
+export const OnboardingDialog = () => {
+  const { VEGA_ENV } = useEnvironment();
+  const [, setOnboardingViewed] = useLocalStorage(
+    constants.ONBOARDING_VIEWED_KEY
+  );
+  const navigate = useNavigate();
+  const browseMarkets = () => {
+    const link = Links[Routes.MARKETS]();
+    navigate(link);
+    setOnboardingViewed('true');
+  };
+  const lead =
+    VEGA_ENV === 'MAINNET'
+      ? t('Start trading on the worlds most advanced decentralised exchange.')
+      : t(
+          'Free from the risks of real trading, Fairground is a safe and fun place to try out Vega yourself with virtual assets.'
+        );
+  return (
+    <div className="flex flex-col pb-2">
+      <div className="flex gap-8">
+        <div className="w-1/2 flex flex-col justify-between pt-3">
+          <ul className="ml-0">
+            <li className="my-3 flex gap-3 text-vega-clight-50 dark:text-vega-cdark-50">
```

Comment:
I added a TW alias for the default text colors, see styles.css

```suggestion
            <li className="my-3 flex gap-3 text-default">
```

Reply:
done